cpu/cortexm_common/include/cpu.h

```diff
@@ -114,6 +187,26 @@ static inline void cortexm_isr_end(void)
     }
 }
 
+/**
+ *  @brief   Branch the execution to a specified address, usually used to
+ *           jump to another RIOT instance.
+ */
+static inline void cortexm_branch_address(uint32_t destination_address)
```

Comment:
Does this work for you on samr21-xpro? I'm getting a compile error in line 205. Also the jumping seems to hard fault.